bodhi/server/scripts/approve_testing.py

```diff
@@ -100,13 +102,18 @@ def main(argv=sys.argv):
             # not reached the karma threshold.
             if update.meets_testing_requirements:
                 print(f'{update.alias} now meets testing requirements')
-                text = str(
-                    config.get('testing_approval_msg') % update.mandatory_days_in_testing)
+
+                text = str(config.get('testing_approval_msg') % update.mandatory_days_in_testing)
                 update.comment(db, text, author=u'bodhi')
 
                 notifications.publish(
                     topic='update.requirements_met.stable',
-                    msg=dict(update=update))
+                    msg=dict(update=update)
+                )
+                if update.autopush:
```

Comment:
We should check that the requested number of days was reached.

Reply:
Let's ensure that there are tests that check all of these conditions:

0. The update has not reached mandatory days in testing or stable_days.
1. The update has reached mandatory days in testing but not stable_days.
2. The update has reached mandatory days in testing and stable_days.
3. All of the above but autopush/autotime is False.

Reply:
Ha yes I used to get this for free in `meets_testing_requirements` but with the last change I need to do the checks.